controller-server/src/main/java/com/yahoo/vespa/hosted/controller/restapi/application/ApplicationApiHandler.java

```diff
@@ -2035,7 +2036,7 @@ private HttpResponse createInstance(String tenantName, String applicationName, S
         if (controller.applications().getApplication(applicationId).isEmpty())
             createApplication(tenantName, applicationName, request);
 
-        controller.applications().createInstance(applicationId.instance(instanceName));
+        controller.applications().createInstance(applicationId.instance(instanceName), Tags.empty());
```

Comment:
Hmm ... supporting `dev/perf` here isn't trivial, and perhaps not that useful?

Reply:
I assumed not.